tools/nnpackage_tool/model2nnpkg/README.md

```diff
@@ -4,20 +4,34 @@
 
 It takes `modelfile` as input and generates `nnpackage`.
 
+## Prerequisite
+
+Install jq
+```
+$ sudo apt-get install jq
+```
+
```

Comment:
`model2nnpkg.sh` is used on compiler's command `one-pack`.
This dependency make additional requirement to compiler.

Reply:
It's too difficult to parse json without `jq` in `model2nnpkg`.
If I cannot use `jq` in `model2nnpkg`, it looks like I'll have to reconfigure `MENIFEST` file outside without modifying this script.